iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -292,4 +295,12 @@ public static void performMetadataDelete(Table icebergTable, String branchName,
     }
     deleteFiles.deleteFromRowFilter(exp).commit();
   }
+
+  public static String convertToHiveType(Type icebergType) {
```

Comment:
should we use HiveSchemaConverter.convertType?

Reply:
Reused `HiveSchemaUtil.convert()`. Done.